spec/linter-codeclimate-spec.js

```diff
@@ -35,4 +37,21 @@ describe('The codeclimate provider for Linter', () => {
       ),
     ),
   );
+
+  it('runs all configured engines', () =>
```

Comment:
Agree. The point of this test is to run an engine that was not in the hardcoded ones, and consequently couldn't be ran before. 

I'm open for suggestions both for the spec description and better specs.